Microsoft.DotNet.Interactive/ExtensionLoaded.cs

```diff
@@ -0,0 +1,18 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using Microsoft.DotNet.Interactive.Commands;
+using Microsoft.DotNet.Interactive.Events;
+
+namespace Microsoft.DotNet.Interactive
+{
+    public class ExtensionLoaded : KernelEventBase
+    { 
+        public ExtensionLoaded(string extensionPath)
```

Comment:
If we think about display feedback, I wonder if something a little richer than a string would be useful. What other use cases might there be for this event? What can subscribers do with a string?